src/core/lib.lua

```diff
@@ -730,12 +730,16 @@ function random_bytes_from_math_random (count)
    return bytes
 end
 
+local lastseed=0
 function randomseed (seed)
    seed = tonumber(seed)
    if seed then
-      local msg = 'Using deterministic random numbers, SNABB_RANDOM_SEED=%d.\n'
-      io.stderr:write(msg:format(seed))
       -- When setting a seed, use deterministic random bytes.
+      if seed ~= lastseed then
+        lastseed = seed
+        local msg = 'Using deterministic random numbers, SNABB_RANDOM_SEED=%d.\n'
+        io.stderr:write(msg:format(seed))
+      end
```

Comment:
I don't think this is necessary.  `lib.randomseed` should be called only once in a process's lifetime.  I think the multiple printouts that you see in the logs correspond to many subprocesses.